native/utils.js

```diff
@@ -0,0 +1,14 @@
+import { IPC_CLIPBOARD_WRITETEXT, IPC_DIALOG_SHOWOPENDIALOG } from "./types.js";
```

Comment:
@xla does `utils.js` make sense to you as a name for this? I don't think these kinds of helper functions belong in `main`; was also thinking about calling it `lib` because it's a library.

Reply:
Alternatively there could be a `native/index.js`

Reply:
I usually find utils/helpers and friends an invitation to become catch all modules for everything that didn't find a proper home. Having it `index` sounds alright, More aptly in his case all these methods are concerned with ipc so that might be an appropriate name for the file.